compiler/code-gen/files/vars-cpp.cpp

```diff
@@ -77,6 +100,7 @@ static void compile_vars_part(CodeGenerator &W, const std::vector<VarPtr> &vars,
   for (auto var : vars) {
     if (!G->settings().is_static_lib_mode() || !var->is_builtin_global()) {
       includes.add_var_signature_depends(var);
+      add_dependent_includes(var->init_val, includes);
```

Comment:
Isn't it better to add this method to `IncludesCollector`, not as a standalone function?
```
includes.add_vertex_depends(var->init_val);
```